go/test/endtoend/reparent/utils_test.go

```diff
@@ -0,0 +1,538 @@
+/*
```

Comment:
nit: utils_test -> test_utils?

Reply:
https://tip.golang.org/cmd/go/#hdr-Test_packages   
Test related files have to end with _test.go, both to access other test files in the package and so as to not get compiled into the regular binary.